octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,15 @@
+package org.example.calculator;
+
+public class Calculator {
+
+    public static void main(String[] args) {
+        int a = 1;
+        int b = 2;
+
+        int c = a + b;
+
+        System.out.println("Sum of a and b is: " + c);
+
```

Comment:
[Severity: Minor] There is no error handling or input validation in the current implementation.
Consider implementing basic error handling to manage potential exceptions (e.g., invalid input types if user input is added).
Rationale: This will improve the robustness of the application and prevent crashes due to unexpected input.
Reference: Best practices for error handling in Java.